### DISCUSSION TITLE
How to increase line height of source code in NetBeans 18?

### DISCUSSION THREAD

**Discussion by gmbhurgri** [apache/netbeans#6027] [Q&A] [answered]
Hi, I want to increase line height between source code lines, or in other words put some padding-bottom and top in each line to make the line separate as the code is too close with default line spacing...

Seems there is no option to change line height or am I missing it? Please guide

**Comment by rtoi** [comment, marked as answer]
Hi,

You can alter the line height by adjusting 'line-height-correction' value in  `org-netbeans-modules-editor-settings-CustomPreferences.xml` file. On my computer (freeBSD) the file is located on `~/.netbeans/18/config/Editors/Preferences` folder.
In that file you should have the following XML  for 'line-height-correction' value.
`<entry javaType="java.lang.Float" name="line-height-correction" xml:space="preserve">
   <value><![CDATA[1.1]]></value>
</entry>` 

By increasing or decreasing the value (1.1 in the example above), you can adjust the line height as you wish.

**Reply by Nazim231** [reply]
Hi,

I don't have `.netbeans` folder, can you please help me with this

**Reply by mbien** [reply]
its the user directory and the location is platform dependent.

The about window will show you where it is according to the readme:
https://github.com/apache/netbeans?tab=readme-ov-file#log-config-and-cache-locations
